packages/backend/server/routes/stats.get.ts

```diff
@@ -0,0 +1,49 @@
+export default eventHandler(async (event) => {
+  try {
+    const binding = useBinding(event);
+
+    let cursor: string | undefined;
+    let totalObjects = 0;
+    const orgs = new Set<string>();
+    const repos = new Set<string>();
+    const commits = new Set<string>();
+    const packages = new Set<string>();
+
+    const prefix = `${usePackagesBucket.base}:`;
+
+    do {
+      const result = await binding.list({ prefix, cursor });
+      const keys = result.objects.map((obj) => obj.key);
+
+      totalObjects += keys.length;
+
+      for (const key of keys) {
+        const trimmedKey = key.slice(prefix.length);
+        const parts = trimmedKey.split(":");
+        const [org, repo, commit, packageName] = parts;

```

Comment:
remove this unused line, do it just in the same place, no need for two lines to use const [....]